pkg/service/schema/model.go

```diff
@@ -7,37 +7,25 @@ import (
 	"github.com/tbd54566975/ssi-service/internal/keyaccess"
 )
 
-const (
-	Version1 string = "1.0"
-)
-
 type CreateSchemaRequest struct {
-	Author string            `json:"author" validate:"required"`
-	Name   string            `json:"name" validate:"required"`
-	Schema schema.JSONSchema `json:"schema" validate:"required"`
+	Name        string            `json:"name" validate:"required"`
+	Description string            `json:"description,omitempty"`
+	Schema      schema.JSONSchema `json:"schema" validate:"required"`
 
 	// If sign == true, the schema will be signed by the author's private key with the specified KID
-	Sign      bool   `json:"signed"`
-	AuthorKID string `json:"authorKid"`
+	Sign      bool   `json:"sign,omitempty"`
+	Issuer    string `json:"issuer,omitempty"`
+	IssuerKID string `json:"issuerKid,omitempty"`
 }
 
 func (csr CreateSchemaRequest) IsValid() bool {
 	return util.IsValidStruct(csr) == nil
 }
 
 type CreateSchemaResponse struct {
-	ID        string              `json:"id"`
-	Schema    schema.VCJSONSchema `json:"schema"`
-	SchemaJWT *keyaccess.JWT      `json:"schemaJwt,omitempty"`
-}
-
-type VerifySchemaRequest struct {
-	SchemaJWT keyaccess.JWT `json:"schemaJwt"`
-}
-
-type VerifySchemaResponse struct {
-	Verified bool   `json:"verified"`
-	Reason   string `json:"reason,omitempty"`
+	ID               string            `json:"id"`
+	Schema           schema.JSONSchema `json:"schema"`
+	CredentialSchema *keyaccess.JWT    `json:"credentialSchema,omitempty"`
```

Comment:
I don't think this should be returned from the `schema.Service`.

Reply:
going to remove this and save it for the next issue